src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/src/extension.ts

```diff
@@ -0,0 +1,80 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import { spawn } from 'child_process';
+import { acquireDotnetInstall } from './acquireDotnetInstall';
+import { getAvailablePort } from "./getAvailablePort";
+
+let launchDebugProxy: vscode.Disposable;
+let killDebugProxy: vscode.Disposable;
+
+export function activate(context: vscode.ExtensionContext) {
+    const outputChannel = vscode.window.createOutputChannel("Blazor WASM Debug Proxy");
+    const pidsByUrl = new Map<string, number>();
+
+    launchDebugProxy = vscode.commands.registerCommand('ms-blazorwasm-companion.launchDebugProxy', async () => {
+        try {
+            const debuggingPort = await getAvailablePort(9222);
+            const debuggingHost = `http://localhost:${debuggingPort}`;
```

Comment:
Just clarifying and i'm sure it's a silly question but is it ever required for the debug proxy to be https?

Reply:
Nope, both Chrome and the debugging proxy assume a HTTP-connection by default [[ref](https://github.com/dotnet/runtime/blob/master/src/mono/wasm/debugger/BrowserDebugHost/Program.cs#L18)].